docs/content/gs-adding-users-proc.adoc

```diff
@@ -27,9 +27,9 @@ Log in as the `customer-admin` user using the credentials that you received in t
 
 == Viewing Users in the Red Hat Single Sign-On administration console
 
-.Prerequisites
+.Prerequisite
 
-* You have logged into the Red Hat Single Sign-On administration console.
+* You have logged into the Red Hat Single Sign-On administration console using the 'Authenticate through Red Hat Single Sign-On' facility.
```

Comment:
I wasn't likely explicit but the link with text `Authenticate through Red Hat Single Sign-On` is available only in 3scale dashboard login page and not in any other product.

Reply:
Fixed.